kafka/kafka.sh

```diff
@@ -37,25 +37,28 @@ function err() {
 
 function install_and_configure_kafka_server() {
   # Find zookeeper list first, before attempting any installation.
-  local ZOOKEEPER_CLIENT_PORT=$(grep 'clientPort' /etc/zookeeper/conf/zoo.cfg \
+  local zookeeper_client_port
```

Comment:
I was't sure if it would be better to have list of declarations of local variables at the top of the function. WDYT @bsidhom ?

Reply:
This isn't explicitly called out in the style guide, but I prefer having local variables declared as late as possible so we can see their use in context. This is in keeping with the [Java style guide](https://google.github.io/styleguide/javaguide.html#s4.8.2.2-variables-limited-scope).